SU-HSDO/stanford_mrc
# MRCD8-244 slideshow responsiveness

# READY FOR REVIEW

# Summary
- Responsive styles for slideshow
- Arrows are reponsive
- Content overlay drops below image with appropriate color changes

# Needed By (Date)
- End of sprint (3/29)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the slideshow is now responsive
3. Note that slides with long text might have a little weirdness in-between tablet and mobile breakpoints

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-244

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
